src/sas/qtgui/Plotting/Slicers/WedgeSlicer.py

```diff
@@ -175,7 +176,10 @@ def _post_data(self, new_sector=None, nbins=None):
             dxw = sector.dxw
         else:
             dxw = None
-        new_plot = Data1D(x=sector.x, y=sector.y, dy=sector.dy, dx=sector.dx)
+        # And here subtract pi when getting angular data back from sector averaging in
+        # manipulations to get back in the -pi,pi range. Also convert from radians to
+        # degrees for nicer display.
+        new_plot = Data1D(x=(sector.x - np.pi) * 180 / np.pi, y=sector.y, dy=sector.dy, dx=sector.dx)
```

Comment:
At the moment, this angular shift is happening even when plotting against Q. I think this line should go back to having `x=sector.x`, and then the following could be added after line 169:
```python
        if self.averager.__name__ == 'SectorPhi':
            sector.x = (sector.x - np.pi) * 180 / np.pi
```

Reply:
Good catch @ehewins

Reply:
In principle if class.__name__ ==name: is not a good construct I've been told. However there are 2 other uses of it in this one method so just going with it for this PR but left a "todo" comment. I suspect the right answer would be a create_plot method in post_data that handles those 3 specifics?